site/pages/search-results-contextual-en.hbs

```diff
@@ -0,0 +1,104 @@
+---
+{
+	"title": "Search results",
+	"language": "en",
+	"altLangPrefix": "search",
+	"pageType": "search",
+	"pageclass": "page-type-search",
+	"breadcrumb": [
+		{ "title": "Home", "link": "https://www.canada.ca/en.html" }
+	],
+	"dateModified": "2018-12-21",
+	"share": "false",
+	"deptfeature": false
+}
+---
+<!--Note: visual example with styles applied: https://gc-proto.github.io/student-loans/scenarioA/services/search-results-contextual-visual.html-->
+<!--Note 2: this template should only be used for contextual search results from a single institution-->
+<!--Note 3: until radio buttons pattern works, please simply include 'All government of Canada websites' link in the well, under the search box-->
+<div class="col-md-12">
+	<h1 property="name">Search results</h1>
+	<div class="well">
+		<form name="cse-search-box" method="GET" action="/en/sr/srb.html" id="cse-search-box">
+			<label class="wb-inv" for="sch-inp">Search Government of Canada websites</label>
+			<input type="hidden" name="st" value="s" />
+			<input type="hidden" name="s5bm3ts21rch" value="x" />
+			<input type="hidden" name="num" value="10" />
+			<input type="hidden" name="st1rt" value="0" />
+			<input type="hidden" name="langs" value="eng" />
+			<input type="hidden" name="cdn" value="canada" />
+			<input type="hidden" name="hq" value="" />
```

Comment:
Do I am right to assume those ```input[type=hidden]``` are very specific to our search tool implementation. If so, I suggest to remove it from this template.

Reply:
No idea. More of a question for Cristian.

I assume you're right though! I'll remove them from the template